src/ir/auth_logic/lowering_ast_datalog.cc

```diff
@@ -269,40 +263,39 @@ std::vector<datalog::DLIRAssertion> LoweringToDatalogPass::SaysAssertionToDLIR(
   return ret;
 }
 
```

Comment:
Could this be turned into a single `MapIter` call?

Reply:
MapIter doesn't work well with `move`?

Reply:
Good point, forgot about that.